airflow/config_templates/airflow_local_settings.py

```diff
@@ -15,242 +15,15 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
-"""Airflow logging settings"""
+"""This module is deprecated. Please use `airflow.logging_config` instead"""
 
-import os
-from typing import Any, Dict, Union
-from urllib.parse import urlparse
+import warnings
 
-from airflow.configuration import conf
-from airflow.exceptions import AirflowException
-from airflow.utils.file import mkdirs
+# pylint: disable=unused-import
+from airflow.logging_config import AwsLambdaHook  # noqa
```

Comment:
This doesn't look right

Reply:
```suggestion
from airflow.logging_config import DEFAULT_LOGGING_CONFIG  # noqa
```

Reply:
Whoops!

Was in a rush to get this finished before heading off for the afternoon